functional_programming_tests/test/launch/with_parameters.test.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+import launch
+from launch_ros.actions import Node
+import launch_testing
+import unittest
+
+
+def generate_test_description():
+
+    with_parameters_test = Node(
+        executable=launch.substitutions.PathJoinSubstitution(
+            [
+                launch.substitutions.LaunchConfiguration("test_binary_dir"),
+                "with_parameters",
```

Comment:
is "with_parameters" the name of the test or does that indicate something else?

Reply:
it's the name of the gtest executable

Reply:
it has to match https://github.com/PickNikRobotics/ros_testing_templates/pull/29/files#diff-f99c709833b5bc59ed71a80dd166b5761a244238bf7f44f904551b9b8f9b3f5aR30